### PR TITLE
Fixes Issue #3821

### PR DESCRIPTION
**Pull request by san7ket**
Declared the constants in constants.py for the base images
Currently supports rhel68 and rhel72
Updates all instances , with the new constants DISTRO_RHEL6, DISTRO_RHEL7 respectively.

### CODE REVIEW

**Comment by omaciel** [comment]
@san7ket thanks for doing this. Can you `--amend` your commit and add a more verbose comment to it? The commit message is very terse right now :/ Check this example: https://github.com/erlang/otp/wiki/writing-good-commit-messages

**Comment by omaciel** [comment]
@kbidarkar @san7ket since this commit will only affect our current/new automation efforts, do we want to backport it into older branches?

File: robottelo/vm.py [hunk 1]
```diff
@@ -15,14 +15,12 @@
 
 from robottelo import ssh
 from robottelo.config import settings
+from robottelo import DISTRO_RHEL6, DISTRO_RHEL7
```
Comment (omaciel) [inline robottelo/vm.py]: `from robottelo.constants`

Comment (san7ket) [inline robottelo/vm.py]: ohh , and I am searching , what went wrong. Thanx :+1:

**Comment by kbidarkar** [comment]
Yes, good point, we do want it :)
1) we need to back port it to other branches too, atleast for 6.2.z branch to start with.
2) And also for 6.1.z branch too

**Comment by omaciel** [comment]
@san7ket you need to actually use `git ci --amend` to change the commit message and not update it via Github ui :)

**Comment by oshtaier** [comment]
The very fundamental class was affected, so it is necessary to provide test results for at least one test per each module that you changed

ACK pending test results

**Comment by kbidarkar** [comment]
@san7ket, I think I have a way to provide the test results for this in an easier way, let's discuss this tomorrow.

**Comment by sghai** [comment]
Looks good. However ACK pending on @omaciel & @oshtaier comments

**Comment by sghai** [comment]
@san7ket : what's pending on this ?

**Comment by san7ket** [comment]
@sghai the test's are failing, need to investigate.

**Comment by san7ket** [comment]
I ran tier3 standalone automation, and it seems like the dns issue is fixed. The issue was with the instances , not honoring the dns entry from the base image.

From the tests attached , can confirm that hosts are able to talk to satellite and this PR will help maintain consistency with images.

[UI HostUnificationTestCase.txt](https://github.com/SatelliteQE/robottelo/files/500189/UI.HostUnificationTestCase.txt)
[CLI HostUnification test run.txt](https://github.com/SatelliteQE/robottelo/files/500190/CLI.HostUnification.test.run.txt)

**Comment by oshtaier** [comment]
Could you please run few tests, let's say:
```
test_activationkey.py:
test_positive_usage_limit

test_contenthost.py:
test_negative_register_twice

ui/test_errata.py:
test_positive_apply_for_host

ui/test_hostunification.py:
test_positive_register_host_via_ak
```
and provide results explicitly there
If you already have the reports just take results from it and put it here

**Comment by san7ket** [comment]
yep , was cleaning the results.

[CLI HostUnification test run.txt](https://github.com/SatelliteQE/robottelo/files/500277/CLI.HostUnification.test.run.txt)
[UI HostUnificationTestCase.txt](https://github.com/SatelliteQE/robottelo/files/500276/UI.HostUnificationTestCase.txt)

**Comment by oshtaier** [comment]
Can you provide it in any other format? as txt doesn't seem readable

**Comment by oshtaier** [comment]
ACK

**Comment by omaciel** [comment]
@san7ket was this cherrypicked into `6.2.x` branch?

**Comment by lpramuk** [comment]
@omaciel nope

**Comment by lpramuk** [comment]
I'll do it shortly

**Comment by lpramuk** [comment]
https://github.com/SatelliteQE/robottelo/pull/3900 ready to merge

**Comment by lpramuk** [comment]
[6.1.z] PR https://github.com/SatelliteQE/robottelo/pull/3901 ready to merge too :-)

**Pull request merged**